itests/eth_block_hash_test.go

```diff
@@ -26,30 +26,36 @@ import (
 func TestEthBlockHashesCorrect_MultiBlockTipset(t *testing.T) {
 	// miner is connected to the first node, and we want to observe the chain
 	// from the second node.
-	blocktime := 250 * time.Millisecond
+	blocktime := 100 * time.Millisecond
```

Comment:
This is reverting a change I made a while back while attempting to deflake this before.